spec/services/invoices/calculate_fees_service_spec.rb

```diff
@@ -181,6 +183,54 @@
             end
           end
         end
+
+        context 'when calculating fees for draft invoice' do
+          let(:invoice) do
+            create(
+              :invoice,
+              :draft,
+              organization:,
+              currency: 'EUR',
+              issuing_date: Time.zone.at(timestamp).to_date,
+              customer: subscription.customer
+            )
+          end
+
+          context 'when no context is passed' do
+            let(:endpoint) { 'https://api.nango.dev/v1/anrok/draft_invoices' }
+
+            it 'creates fees' do
+              result = invoice_service.call
+
+              aggregate_failures do
+                expect(result).to be_success
+
+                expect(result.invoice.fees_amount_cents).to eq(100)
```

Comment:
Can you please either alter response from Anrok and set different numbers or add new expectation to check which endpoint is called?

Reply:
ah, right, we have allow in the before block, not expect 🤦‍♀️